specreduce/background.py

```diff
@@ -103,18 +107,22 @@ def _to_trace(trace):
 
         if np.any(bkg_wimage > 1):
             raise ValueError("background regions overlapped")
+        if np.any(np.sum(bkg_wimage, axis=self.crossdisp_axis) == 0):
+            raise ValueError("background window does not remain in bounds across entire dispersion axis")  # noqa
```

Comment:
This catches the case where the background is completely out of bounds _at any pixel_ (except I think we should avoid using the word pixel since we'll have wavelength calibration eventually), but not necessarily across the entire image.  So maybe "background window lies completely out of bounds somewhere along the dispersion axis"?